build-image/first-boot.bash

```diff
@@ -151,8 +151,14 @@ sleep 10  # Related to: https://github.com/openhab/openhabian/issues/441#issueco
 echo "OK"
 
 echo -n "$(timestamp) [openHABian] Updating repositories and upgrading installed packages... "
-apt-get --yes --fix-broken install &>/dev/null
-if apt-get --yes upgrade &>/dev/null; then echo "OK"; else echo "FAILED"; fail_inprogress; fi
+apt --yes --fix-broken install &>/dev/null
+apt-get --yes upgrade &>/dev/null
+res=$?
+if [[ $res -eq 100 ]]; then 
+  dpkg --configure -a
+  apt --yes --fix-broken install &>/dev/null
+  if apt-get --yes upgrade &>/dev/null; then echo "OK"; else echo "FAILED"; fi
+fi
```

Comment:
approach itself looks good.

`else echo "OK"` for the normal use case

Reply:
like so?
Rest ok?